samples/philosophers/src/main.c

```diff
@@ -87,7 +87,7 @@
 
 #include "phil_obj_abstract.h"
 
-#define fork(x) (forks[x])
+#define philosopher_fork(x) (forks[x])
```

Comment:
Btw wouldn't this be clearer to have this macro be uppercase? (in other samples too, ofc). In any case we can definitely clean up as a follow-up, since it obviously was lowercase until now and didn't seem to have been an issue for anyone :)